app/views/members/_become_speaker_form.slim

```diff
@@ -0,0 +1,14 @@
+= form_tag become_speaker_path do
+  = text_field_tag :name, current_user&.first_name,
+                          required:    true
+  br
+  br
```

Comment:
I think, we should wait for a design first, yes `br` is not good for formatting form or other things, but in this case is not so bad.